src/app.jl

```diff
@@ -108,29 +136,86 @@ end
 """
 
 function dash(name::String;
-    external_stylesheets ::Vector{String} = Vector{String}(),
-    external_scripts ::Vector{String} = Vector{String}(),
-    url_base_pathname="/",
-    assets_folder::String = "assets")
-        result = DashApp(name,
-            external_stylesheets=external_stylesheets,
-            external_scripts=external_scripts,
-            url_base_pathname=url_base_pathname,
-            assets_folder = assets_folder
-            )
+        external_stylesheets = ExternalSrcType[],
+        external_scripts  = ExternalSrcType[],
+        url_base_pathname = nothing,        
```

Comment:
@rpkyle See the comment to the previous issue
Also
> I suspect that this was never updated

This is why I don't like to write expanded docstrings in the initial stages of development. Because they need to be completely rewritten before release, or they will accumulate similar inconsistencies. With frequent changes to an api that has not yet stabilized it is very difficult to track that there are no inconsistencies in the code and documentation